### PR TITLE
[SDK]enable telemetry by default

### PR DESCRIPTION
**Pull request by D-W-**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

This PR changed:
- Default enable telemetry, user can disable by running `pf config set cli.telemetry_enabled=false` since it's green light by privacy team.
- Removed Europe logger, since we only contain System Metadata in telemetry

This pull request includes several changes to the telemetry functionality in the codebase, including updates to logging, configuration, and test fixtures. The most important changes include updating the telemetry functionality to only disable if the user explicitly sets it to false, and removing unused constants and methods related to EU user configuration.

Telemetry improvements:

* <a href="diffhunk://#diff-3cfb982f5d43767ea1a397647af019c0cb5cd5b1c3c5340097c819e6e86a2eefL74-R69">`src/promptflow/promptflow/_telemetry/telemetry.py`</a>: Updated telemetry functionality to only disable if the user explicitly sets it to false, and cleaned up the code by removing unused constants and reusing log handlers only if telemetry is enabled. (is_telemetry_enabled, telemetry.py, <a href="diffhunk://#diff-3cfb982f5d43767ea1a397647af019c0cb5cd5b1c3c5340097c819e6e86a2eefL74-R69">src/promptflow/promptflow/_telemetry/telemetry.pyL74-R69</a>)
* <a href="diffhunk://#diff-d80d70a0879d410337d9905c070a80049e5fa2808ba4f4089c78625067ae32d7L29-L31">`src/promptflow/promptflow/_telemetry/logging_handler.py`</a>: Removed the `eu_user` attribute and parameter from `PromptFlowSDKLogHandler`'s constructor and the `is_eu_user()` method, and removed the `if config.is_eu_user():` block from `get_appinsights_log_handler` function, making the instrumentation key used for telemetry independent of the user's location. (PromptFlowSDKLogHandler, get_appinsights_log_handler, <a href="diffhunk://#diff-d80d70a0879d410337d9905c070a80049e5fa2808ba4f4089c78625067ae32d7L29-L31">src/promptflow/promptflow/_telemetry/logging_handler.pyL29-L31</a>)
* <a href="diffhunk://#diff-b2ec38c5870b7faa23ef68ce3076a71978711f94d9877060dea64451e9bc747fL12-R13">`src/promptflow/tests/sdk_cli_azure_test/e2etests/test_telemetry.py`</a>: Added assertions to ensure correct telemetry logging, refactored import statement, and updated function to set extension telemetry consent configuration instead of EU user configuration. (TestTelemetry.test_logging_handler, <a href="diffhunk://#diff-b2ec38c5870b7faa23ef68ce3076a71978711f94d9877060dea64451e9bc747fL12-R13">[1]</a> <a href="diffhunk://#diff-b2ec38c5870b7faa23ef68ce3076a71978711f94d9877060dea64451e9bc747fL35-R45">[2]</a>

Configuration improvements:

* `src/promptflow/promptflow/_sdk/_configuration.py`: Removed the `is_eu_user()` method and the `EU_USER` and `EXTENSION_EU_USER` constants from the `Configuration` class since they were not being used. (Configuration)
* `src/promptflow/CHANGELOG.md`: Telemetry is now enabled by default and can be disabled using the `cli.telemetry_enabled` configuration. (src/promptflow/CHANGELOG.md)
* <a href="diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L169-R171">`README.md`</a>: Updated to reflect that telemetry is now on by default and users can opt out if they don't want to participate.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: src/promptflow/promptflow/_telemetry/telemetry.py [hunk 1]
```diff
@@ -46,19 +44,15 @@ def _get_telemetry_values(self, *args, **kwargs):  # pylint: disable=unused-argu
 
 
 def is_telemetry_enabled():
-    """Check if telemetry is enabled. User can enable telemetry by
-    1. setting environment variable TELEMETRY_ENABLED to true.
-    2. running `pf config set cli.telemetry_enabled=true` command.
-    If None of the above is set, telemetry is disabled by default.
+    """Check if telemetry is enabled. Telemetry is enabled by default.
+    User can disable it by:
+    1. running `pf config set cli.telemetry_enabled=false` command.
```
Comment (wangchao1230) [inline src/promptflow/promptflow/_telemetry/telemetry.py:49]: cli.telemetry_enabled ->  telemetry.enabled

File: README.md [hunk 2]
```diff
@@ -166,11 +166,9 @@ use of the software operates as your consent to these practices.
 
 ### Telemetry Configuration
 
-Telemetry collection is off by default.
+Telemetry collection is on by default.
 
-To opt in, please run `pf config set cli.telemetry_enabled=true` to turn it on.
-
-For Europe user, please run `pf config set cli.eu_user=true` to make sure telemetry is sent to Europe server.
+To opt out, please run `pf config set cli.telemetry_enabled=false` to turn it off.
```
Comment (wangchao1230) [inline README.md:171]: Aslo add sth like this: 
image

Comment (D-W-) [reply README.md:171]: Added in `pf`
1st time run:
image

2nd time run:
image

Comment (wangchao1230) [inline README.md:171]: telemetry.enabled

Comment (wangchao1230) [reply README.md:171]: put the ascii art to top

Comment (D-W-) [reply README.md]: Change to this
image

**Review by wangchao1230 — APPROVED** [review]

**Review by crazygao — APPROVED** [review]

**Pull request merged**